### DISCUSSION TITLE
Unable to resolve Microsoft.UI.Xaml.Application (Reason: None, Location:Unknown, Candidates: )	Weather.History.Wasm

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#9132] [Q&A]
### Current behavior

```<prism:PrismApplication
    x:Class="Weather.History.App"
    xmlns:prism="using:Prism.DryIoc"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

    <Application.Resources>
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
    </Application.Resources>
</prism:PrismApplication>```

Prismapplication cannot be resolved though it is found in codebehind. 






### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

  <ItemGroup>
    <PackageReference Include="DryIoc.dll" Version="4.8.8" />
    <PackageReference Include="LiveChartsCore.SkiaSharpView.Uno" Version="2.0.0-beta.300" />
    <PackageReference Include="Microsoft.Windows.Compatibility" Version="7.0.0-preview.5.22301.12" />
    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0-preview.5.22301.12" />
    <PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.4.0" />
    <PackageReference Include="Uno.UI.WebAssembly" Version="4.4.0-dev.331" />
    <PackageReference Include="Uno.UI.RemoteControl" Version="4.4.0-dev.331" Condition="'$(Configuration)'=='Debug'" />
    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.4.0-dev.331" />
    <PackageReference Include="Uno.Wasm.Bootstrap" Version="4.0.0-dev.172" />
    <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="4.0.0-dev.172" />
  </ItemGroup>

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

Version 17.2.4

### Relevant plugins

_No response_

### Anything else we need to know?

public abstract class PrismApplication : PrismApplicationBase is found in // C:\Users\gombata-locadmin\source\repos\Weather.History\Prism.NetStandard\Prism.DryIoc.Uno.dll
// Decompiled with ICSharpCode.Decompiler 7.1.0.6543

In Xaml file not found. 

It is not possible to paste screen shots in any of these input boxes.

**Comment by sigmarsson** [comment]
![image](https://user-images.githubusercontent.com/43340959/175528881-b81741f4-c0e0-4d10-9fe7-fe61a8278b66.png)

The Prism class is found and located but not found in ```App.xaml```

**Reply by jeromelaban** [reply]
Prism is still not supporting Uno 4.x, the answer has not changed since https://github.com/unoplatform/uno/discussions/8593#discussioncomment-2614998.

**Reply by sigmarsson** [reply]
@jeromelaban Thank you. I think neither ```Prism``` master's head nor the latest released nuget package is eligible for ```Uno 4.x```. I built it by myself but to no avail. 

I have this resepective assembly version :

![image](https://user-images.githubusercontent.com/43340959/175772073-97c007c0-4481-4c9d-a038-41021e319838.png)

1. How much time shall I wait until I can try for size my current Windows Desktop head's performance with Web Assembly powered by Uno Platform ? 
2. Or do you recommend to phase out ```Prism``` because 
   a) it has dropped off of .NET Foundation
   b) nuget distribution has stopped ?

**Reply by jeromelaban** [reply]
The master of Prism is [updated to Uno 4.3.8](https://github.com/PrismLibrary/Prism/blob/master/Packages.props#L14-L15), but the Prism team still has to publish an update for their packages. It would still be best to ask for an update on this topic on the official repo.

**Reply by jeromelaban** [reply]
If you need more support on this, you can see their developer support [here](https://avantipoint.com/developer/support), or [here](https://github.com/sponsors/dansiegel).
